docs/en-US/component/badge.md

```diff
@@ -51,16 +51,16 @@ badge/dot
 
 ### Badge Attributes
 
-| Name     | Description                                                                     | Type                                                        | Default    |
-| -------- | ------------------------------------------------------------------------------- | ----------------------------------------------------------- | ---------- |
-| `value`  | display value.                                                                  | `string \| number`                                          | `''`       |
-| `max`    | maximum value, shows `{max}+` when exceeded. Only works if `value` is a number. | `number`                                                    | `99`       |
-| `is-dot` | if a little dot is displayed.                                                   | `boolean`                                                   | `false`    |
-| `hidden` | hidden badge.                                                                   | `boolean`                                                   | `false`    |
-| `type`   | badge type.                                                                     | `'primary' \| 'success' \| 'warning' \| 'danger' \| 'info'` | `'danger'` |
+| Name   | Description                                                                   | Type                                                               | Default |
+| ------ | ----------------------------------------------------------------------------- | ------------------------------------------------------------------ | ------- |
+| value  | display value.                                                                | ^[string] / ^[number]                                              | ''      |
+| max    | maximum value, shows `{max}+` when exceeded. Only works if value is a number. | ^[number]                                                          | 99      |
+| is-dot | if a little dot is displayed.                                                 | ^[boolean]                                                         | false   |
+| hidden | hidden badge.                                                                 | ^[boolean]                                                         | false   |
+| type   | badge type.                                                                   | ^[enum]`'primary' \| 'success' \| 'warning' \| 'danger' \| 'info'` | danger  |
 
 ### Badge Slots
```

Comment:
## Suggest

`### Badge Slots` -> `### Slots`

Reply:
Thank you @ryuhangyeong 👍